getting-started/vs-integration/convert-project-wizard.md

```diff
@@ -57,18 +57,28 @@ Alternatively, instead of the context menu, you can use the VS **Extensions** me
 
 ![Start the Convert Project Wizard](images/vs-ext-convert-project-entry.png)
 
-## Features
+## Converting the Project Thought the Wizard
```

Comment:
The article is all about the wizard so you can afford to skip the word "Wizard" in this heading for brevity.
```suggestion
## Converting the Project
```